tests/parsers/test_pw.py

```diff
@@ -29,9 +29,8 @@ def _generate_inputs(calculation_type='scf', settings=None, metadata=None):
     return _generate_inputs
 
 
-def test_pw_default(
-    aiida_profile, fixture_localhost, generate_calc_job_node, generate_parser, generate_inputs, data_regression
-):
+@pytest.mark.usefixture('aiida_profile')
```

Comment:
Just so you're aware, I think `pytestmark` can apply the fixture to all tests in a test module (haven't tried this, only looked at the documentation): https://docs.pytest.org/en/stable/reference.html#globalvar-pytestmark

Reply:
How does this work though? Following the docs, I added `pytestmark = pytest.mark.usefixture('aiida_profile')` at the top of the module file, but that gives me a bunch of warnings:
```
tests/parsers/test_q2r.py:13
  /home/sph/code/aiida/env/dev/aiida-quantumespresso/tests/parsers/test_q2r.py:13: PytestUnknownMarkWarning: Unknown pytest.mark.usefixture - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.usefixture('aiida_profile')

```

Reply:
Seems it's called `pytest.mark.usefixtures`, not `pytest.mark.usefixture`. I have no idea why it worked without the `pytestmark = ...` though.

Reply:
Ah, the `aiida_profile` fixture is used by default when using the AiiDA pytest plugin. We _could_ just remove all those usefixture marks:

https://github.com/aiidateam/aiida-core/blob/1e1bdf2dee779970654ed9b22eb996b78e9c4149/aiida/manage/tests/pytest_fixtures.py#L27